generated/nifake/nifake/unit_tests/test_converters.py

```diff
@@ -294,6 +294,92 @@ def test_repeated_capabilities_invalid_resource_names():
     assert test_result_list == '0/1:2,'
 
 
+def test_convert_single_group_repeated_capabilities():
+    test_result_list = _converters.convert_single_group_repeated_capabilities('0')
+    assert test_result_list == ['0']
+    test_result_list = _converters.convert_single_group_repeated_capabilities('0-2')
+    assert test_result_list == ['0', '1', '2']
+    test_result_list = _converters.convert_single_group_repeated_capabilities('0:2')
+    assert test_result_list == ['0', '1', '2']
+    test_result_list = _converters.convert_single_group_repeated_capabilities('Dev1/0')
+    assert test_result_list == ['Dev1/0']
+    test_result_list = _converters.convert_single_group_repeated_capabilities('Dev1/0-2')
+    assert test_result_list == ['Dev1/0', 'Dev1/1', 'Dev1/2']
+    test_result_list = _converters.convert_single_group_repeated_capabilities('Dev1/0:2')
+    assert test_result_list == ['Dev1/0', 'Dev1/1', 'Dev1/2']
```

Comment:
we're missing tests with random but valid whitespace.
we're missing tests with different capitalization.